### ISSUE TITLE
edit venue: has no choice to show a map

### ISSUE THREAD

**Issue by jojo12** [jemproject/JEM-Project#1556]
When I create in frontend a new venue, there is no button to choose a map (like in backend). 
So, there is no map or link to a map shown. (nor Google nor OSM)
This is only, when I choose this button in backend.

**Comment by jojo12** [comment]
I continued to test: this fieldset is shown only, when JEM settings is google map. But it seems, this button is needed too for showing osm!

**Comment by Heklaterriol** [comment]
> The conditions need more than ` if ($global_show_mapserv == 2 || $global_show_mapserv == 3) : ?> `

correct, it is fixed with `if ($global_show_mapserv == 2 || $global_show_mapserv == 3  || $global_show_mapserv == 5) : ?> `

**Comment by Heklaterriol** [comment]
please check and confirm the fix and close the issue

**Comment by jojo12** [comment]
Sorry, I tested and it works fine.
